pkg/controller/v1beta1/inferenceservice/components/predictor.go

```diff
@@ -223,6 +267,37 @@ func (p *Predictor) Reconcile(isvc *v1beta1.InferenceService) error {
 		}
 		isvc.Status.PropagateStatus(v1beta1.PredictorComponent, status)
 	}
-
+	statusSpec, _ := isvc.Status.Components[v1beta1.PredictorComponent]
+	if rawDeployment {
+		podLabelValue = "isvc." + isvc.ObjectMeta.Name + "-predictor-default"
```

Comment:
Let's use the util function [DefaultPredictorServiceName](https://github.com/kserve/kserve/blob/master/pkg/constants/constants.go#L360)

Reply:
@yuzisun Updated, Thanks!